src/app/pokemon/components/pokemons-modal/pokemons-modal.component.css

```diff
@@ -0,0 +1,59 @@
+:root {
+  --backGgroundColor: rgb(98, 106, 112);
+  --textColor: rgb(241, 242, 242);
+  --buttonColor: rgba(255, 140, 0, 0.992);
+  --blackBorder: rgba(255, 255, 255, 0);
+  --red: rgb(255, 0, 0);
+}
```

Comment:
variables are repeated, they should be in only one css

Reply:
unify all in global style.css